dattri/func/projection.py

```diff
@@ -457,7 +457,7 @@ def free_memory(self) -> None:
         del self.ch_input
         self.input_allocated = False
 
-    def project(self, features: Union[dict, Tensor], ensemble_id: int) -> Tensor:
+    def dict_project(self, features: Union[dict, Tensor], ensemble_id: int) -> Tensor:
```

Comment:
will this `dict_project` be used later?

Reply:
yeah, probably. If we are going to support dict input